src/sass/components/_footer.scss

```diff
@@ -1,67 +1,105 @@
-.m-footer {
-  flex-shrink: 0;
-  background-color: var(--footer-background-color);
+.footer {
+  background-color: #003262;
+  padding: 15px;
 }
 
-.m-footer__content {
-  color: $white;
-  text-align: center;
-  padding: 30px $mobile-space;
+.footer-main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
 
-  @include respond-to('medium') {
-    padding: 70px 0;
-    max-width: 330px;
-    margin: 0 auto;
-  }
+.footer-left {
+  margin: 5px;
+}
+
+.footer-title, .footer-subtitle {
+  font-weight: bolder;
+  color: #fff;
+}
+
+.footer-title {
+  font-size: 24px;
 }
 
-.m-footer-copyright {
-  font-size: 0.875rem;
-  letter-spacing: 0.3px;
-  line-height: 1.75rem;
-  margin-bottom: $mobile-space;
+.footer-subtitle {
+  font-size: 16px;
+}
+
+.footer-social {
+  margin: 15px 0px;
+}
 
-  span {
-    display: block;
+.footer-social a {
+  font-size: 25px;
+  color: #fff;
+  margin-right: 15px;
+}
 
-    @include respond-to('medium') {
-      display: inline;
-    }
+.footer-social a:last-of-type {
+  margin-right: 0px;
+}
 
-    &:nth-child(2) {
-      display: none;
+.footer-mailing-list button {
+  background-color: transparent;
+  color: #fff;
+  font-size: 16px;
+  font-weight: bolder;
+  text-decoration: none;
+  padding: 10px;
+  border-color: white;
 
-      @include respond-to('medium') {
-        display: inline;
-      }
-    }
-  }
+  transition-duration: 0.3s;
+}
 
-  &.jslicense {
-    margin-bottom: 0;
-  }
+.footer-mailing-list button:hover {
+  background-color: white;
+  color: #003262;
+  cursor: pointer;
 }
 
-.m-footer-social {
+.footer-right {
   display: flex;
-  align-items: center;
-  justify-content: center;
-  margin-bottom: 20px;
-
-  a {
-    display: inline-block;
-    color: $white;
-    margin: 0 15px;
-
-    span {
-      color: inherit;
-      font-size: 1rem;
-    }
-  }
+  align-items: stretch;
 }
 
-.rtl {
-  .m-footer-social {
-    direction: ltr;
-  }
+.footer-col {
+  width: 50%;
+  flex-grow: 1;
+  display: inline-block;
+}
+
+.footer-col ul {
+  margin: 0px;
+  padding-left: 0px;
+  list-style-type: none;
+}
+
+.footer-col li {
+  margin: 10px 0px;
+}
+
+.footer-col a {
+  display: block;
+  font-size: 18px;
+  font-weight: bolder;
+  color: rgba(253, 180, 21);
```

Comment:
This color is a little warm, the hexcode for the color in the image is `#fed966`